routes/index.js

```diff
@@ -3,35 +3,34 @@ import express from 'express';
 import AuthController from '../controllers/AuthController';
 import UsersController from '../controllers/UsersController';
 import LoansController from '../controllers/LoansController';
-import AppController from '../controllers/AppController';
 
 import AuthenticationMiddleware from '../middleware/authentication';
+import ParamterValidators from '../middleware/validators.js';
 
 const router = express.Router();
 
-/* GET home page. */
-router.get('/', AppController.index);
-router.get('/about', AppController.about);
-
 router.post('/auth/signup',
     AuthenticationMiddleware.generateToken,
```

Comment:
You should not generate token before data validation. You can move this middleware to the bottom of the chain

Reply:
Data validation before token generation. Noted